jena-arq/src/test/java/org/apache/jena/riot/resultset/rw/TestResultSetWriterCSV.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.jena.riot.resultset.rw;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.ByteArrayOutputStream;
+
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.resultset.ResultSetWriter;
+import org.junit.Test;
+
+public class TestResultSetWriterCSV {
+
+	@Test
```

Comment:
Could you replace tabs with space please?

Reply:
Done